internal/security/pipedhexreader/interface.go

```diff
@@ -0,0 +1,15 @@
+//
+// Copyright (c) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package pipedhexreader
+
+// PipedHexReader is an interface to read hex bytes from
+// standard output stream of an executable into a byte array
+type PipedHexReader interface {
+	// ReadHexBytesFromExe invokes executable with args
+	// and reads hex bytes from stdout and returns an array
+	ReadHexBytesFromExe(executablePath string, args []string) ([]byte, error)
```

Comment:
From what I recall, your original proposal had the ability for command-line options to be specified and passed to the IKM binary, however in this PR, while `ReadHexBytesFromExe` includes an args parameter, an empty slice is always used (except for some of the test cases). Why keep this parameter?

Reply:
Good point.  Removed.